test/container/Dockerfile-Void

```
@@ -0,0 +1,43 @@
+FROM ghcr.io/void-linux/void-glibc-full
+
+RUN xbps-install -Syu \
```

```suggestion
RUN xbps-install -Syu xbps && xbps-install -yu \
```
ensure the container build works if xbps needs an update